lib/config/virtual_container_host.go

```diff
@@ -124,6 +124,8 @@ type Container struct {
 	ContainerNameConvention string
 	// Permitted datastore URLs for container storage for this virtual container host
 	ContainerStores []url.URL `vic:"0.1" scope:"read-only" recurse:"depth=0"`
+	// HostSelctionRequired vic to select host if DRS is disabled & ROBO Adv. License
+	HostSelectionRequired bool `vic:"0.1" scope:"read-only"`
```

Comment:
Thought we were going to determine this at boot time in the endpoint?

Reply:
We discussed it and think that's the right thing to do longer term, but for now I was approaching the same way we handle license / feature eval currently...which is only during install.